src/content/reference/manifest.mdx

```diff
@@ -752,6 +763,20 @@ The name of the endpoint.
 
 The url of the endpoint.
 
+> Dynamic endpoints: The url can be set during deployment stage by sharing `$OKTETO_EXTERNAL_{EXTERNAL_NAME}_ENDPOINTS_{ENDPOINT_NAME}_URL`
```

Comment:
Can we add a sample okteto.yml below this?

```suggestion
##### Dynamic endpoints

The `url` of the endpoint can also be set dynamically during the deployment stage. You can use this for endpoints generated as part of your deployment pipeline (e.g. if you create a Cloud Database). 

After creating the resource, set the URL dynamically by setting an environment variable following the pattern below:

`$OKTETO_EXTERNAL_{EXTERNAL_NAME}_ENDPOINTS_{ENDPOINT_NAME}_URL environment variable`


```# okteto.yaml
deploy:
- name: deploy application
  command: kubectl apply -f  my-app.yml
- name: create database in the cloud
  command: | 
      endpoint_url=$(curl https://internal-service/create-db)
      export OKTETO_EXTERNAL_DB_ENDPOINTS_DASHBOARD_URL=$endpoint_url
external:
  db:
    notes: docs/database.md
    endpoints:
    - name: dashboard```

```